content/configuration/externals.md

```diff
@@ -93,7 +93,22 @@ This syntax is used to describe all the possible ways that an external library c
 
 ### function
 
-?> TODO - Not sure how to use it in and function form. Would be great to see a sample.
+It might be useful to define your own function to control the behavior of what you want to externalize from Webpack. [webpack-node-externals](https://www.npmjs.com/package/webpack-node-externals), for example, excludes all modules from the node_modules and provides some options to, for example, whitelist packages.
+
+It basically comes down to this:
+
+```javascript
+externals: [
+  function(context, request, callback) {
+    if (/^yourregex$/.test(request){
```

Comment:
That reminds me, it would be a very good idea to figure out how to lint the code examples (longer check unless we can constrain it to only changed files, but worth it).